app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -29,7 +29,7 @@
           <input type="radio"
                  name="storageClass"
                  ng-model="claim.storageClass">
-            No Storage Class
+            No storage class or use default if set
```

Comment:
This reads a little weird to me, maybe just "Default storage class." and then add a help block underneath that says
"If your administrator has defined a default storage class then that will be used.  Otherwise no storage class will be selected."

@spadgett thoughts?